src/software/jetson_nano/services/motor.cpp

```diff
@@ -334,6 +336,15 @@ MotorService::MotorFaultIndicator MotorService::checkDriverFault(uint8_t motor)
     return MotorFaultIndicator(drive_enabled, motor_faults);
 }
 
+void MotorService::disableVelocity(double current_velocity, double previous_velocity)
```

Comment:
id suggest renaming this to runawayCheck()

Reply:
Removing the function as it is unused